src/main/java/UseCase/editPlaylist.java

```diff
@@ -1,4 +1,41 @@
 package UseCase;
+import Entities.Playlist;
+import Entities.Song;
+import Entities.User;
+
+import java.util.ArrayList;
 
 public class editPlaylist {
+
+
+
+    public boolean isCollaborator(User user, Playlist playlist){
+        ArrayList<User> collabList = playlist.showCollaborators();
+        boolean isCollaborator = false;
+        for (User editor:collabList){
+            if (editor == user){
+                isCollaborator = true;
+            }
+        }
+        return  isCollaborator;
+    }
+    public void removeSong(User user, Song song, Playlist playlist){
+        boolean isCollaborator = isCollaborator(user,playlist);
+        if(playlist.getOwner() == user || isCollaborator)
+            for (Song s: playlist.songList){
```

Comment:
This isn't going to compile! songList is a private attribute, you have to use the getter lol

Reply:
I saw that problem as well. I made a getter to retrieve the collaborators.

Reply:
> This isn't going to compile! songList is a private attribute, you have to use the getter lol

Im going to remove this edit playlist from the current merge, and push the code through the playlist curation branch

Reply:
Makes sense, but the problem is that there's no getter for the songList, not the collaborators. I fixed this in my PR, if you re-merge from main you can use it.

Reply:
> Makes sense, but the problem is that there's no getter for the songList, not the collaborators. I fixed this in my PR, if you re-merge from main you can use it.

Are you talking about showSongList() ? Because if so I already have it

Reply:
Yep, I added it a week ago but the PR was never approved lol, I just fixed all the conflicts, should be good now

Reply:
> Yep, I added it a week ago but the PR was never approved lol, I just fixed all the conflicts, should be good now

Do you know why the github autograder is not working ?